compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5825,50 +5825,13 @@ OMR::Z::TreeEvaluator::aloadEvaluator(TR::Node * node, TR::CodeGenerator * cg)
    return aloadHelper(node, cg, NULL);
    }
 
+/**
+ * Handles TR_Aiadd on 32-bit
+ */
```

Comment:
This comment isn't particularly useful as it is obvious from the function name. Similar comment for the below one on `TR_Aladd`. Do you agree that we should remove it?